tests/test_to_thread.py

```diff
@@ -183,3 +183,10 @@ def exception_handler(loop: object, context: Any = None) -> None:
     asyncio_event_loop.set_exception_handler(exception_handler)
     asyncio_event_loop.run_until_complete(to_thread.run_sync(time.sleep, 0))
     assert not exceptions
+
+
+def test_asyncio_run_sync_multiple(asyncio_event_loop: asyncio.AbstractEventLoop) -> None:
+    """Regression test for #304."""
+    asyncio_event_loop.call_later(0.5, asyncio_event_loop.stop)
+    for _ in range(3):
+        asyncio_event_loop.run_until_complete(to_thread.run_sync(time.sleep, 0))
```

Comment:
Do you need to do an `assert not t.is_alive()` loop like the above tests?

Reply:
I'm not sure what you mean. Where would `t` come from?

Reply:
Ah, right. I didn't even think about that because the point of the test was to not hang as it did without the fix. I'll copy the assertions from the other test.